src/procedures/proc_wcc.c

```
@@ -0,0 +1,357 @@
+	RelationID **rels  // [output] relationships
+) {
+	// expecting configuration to be a map
+	ASSERT(lbls            != NULL);
+	ASSERT(rels            != NULL);
+	ASSERT(SI_TYPE(config) == T_MAP);
+
+	// set outputs to NULL
+	*lbls = NULL;
+	*rels = NULL;
+
+	uint match_fields = 0;
```

Cppcheck (2.10-2)

[error] Memory leak

(memleak)